users/tests/test_views.py

```diff
@@ -231,6 +229,7 @@ def test_user_new_account(self):
         })
 
     def test_user_duplicate_username_email(self):
+        settings.ACCOUNT_PREVENT_ENUMERATION = False
```

Comment:
this may require `override_settings` from the test suite? im not familiar enough with this to know the potential spillage...

Reply:
Better to use `override_settings` as it will undo the setting change at the end of the test, resetting it to its original value. Setting it directly like this might leak to the other tests (I think)